<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="180680" started-at="2017-07-06T19:33:27Z" finished-at="2017-07-06T19:36:27Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="180680" started-at="2017-07-06T19:33:27Z" finished-at="2017-07-06T19:36:27Z">
      <class name="com.vh.ui.tests.PatientNavigationTest">
        <test-method status="PASS" signature="buildUp()[pri:0, instance:com.vh.ui.tests.PatientNavigationTest@2e7157c7]" name="buildUp" is-config="true" duration-ms="98651" started-at="2017-07-06T14:33:27Z" finished-at="2017-07-06T14:35:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buildUp -->
        <test-method status="FAIL" signature="verify_LeftNavigationPane()[pri:1, instance:com.vh.ui.tests.PatientNavigationTest@2e7157c7]" name="verify_LeftNavigationPane" duration-ms="78931" started-at="2017-07-06T14:35:05Z" finished-at="2017-07-06T14:36:24Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed to expand the Patient Admin menu expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed to expand the Patient Admin menu expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.vh.ui.tests.PatientNavigationTest.verify_LeftNavigationPane(PatientNavigationTest.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_LeftNavigationPane -->
        <test-method status="PASS" signature="afterTest(java.lang.reflect.Method, org.testng.ITestResult)[pri:0, instance:com.vh.ui.tests.PatientNavigationTest@2e7157c7]" name="afterTest" is-config="true" duration-ms="545" started-at="2017-07-06T14:36:24Z" finished-at="2017-07-06T14:36:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.vh.ui.tests.PatientNavigationTest.verify_LeftNavigationPane() throws com.vh.ui.exceptions.WaitException,com.vh.ui.exceptions.URLNavigationException,java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[TestResult name=verify_LeftNavigationPane status=FAILURE method=PatientNavigationTest.verify_LeftNavigationPane()[pri:1, instance:com.vh.ui.tests.PatientNavigationTest@2e7157c7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.vh.ui.tests.PatientNavigationTest@2e7157c7]" name="tearDown" is-config="true" duration-ms="2459" started-at="2017-07-06T14:36:25Z" finished-at="2017-07-06T14:36:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.vh.ui.tests.PatientNavigationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
